ptvsd/wrapper.py

```diff
@@ -790,6 +791,30 @@ def process_launch_arguments(self):
             redirect_output = ''
         self.pydevd_request(pydevd_comm.CMD_REDIRECT_OUTPUT, redirect_output)
 
+    def __parse_launch_option_value(self, value):
+        if value == 'True' or value == 'False':
```

Comment:
We need to add support for attaching as well. Or we could make that a separate PR.

Reply:
I will handle attach separately: https://github.com/Microsoft/ptvsd/issues/248